src/pages/Login/Login.jsx

```diff
@@ -1,8 +1,103 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
+import { useState } from 'react';
+import '../../components/Button/Button.scss';
+import '../../styles/variable.scss';
 
 const Login = () => {
-  return <div>login</div>;
+  const [userID, saveUserID] = useState('');
+  const [userPassword, saveUserPassword] = useState('');
+  const navigate = useNavigate();
+  const [isClicked, setIsClicked] = useState(false);
+
+  const goToMain = () => {
+    navigate('/main');
+  };
+  // const goSignup = () => {
+  //   navigate('/Signup');
+  // };
+
+  // const isInputValid = userEmail.includes('@') && userPassword.length >= 5;
+
+  // const doFetch1 = () => {
+  //   fetch('http://10.58.52.202:8000/users/signin', {
+  //     method: 'POST',
+  //     headers: {
+  //       'Content-Type': 'application/json;charset=utf-8',
+  //     },
+  //     body: JSON.stringify({
+  //       email: userEmail,
+  //       password: userPassword,
+  //     }),
+  //   })
+  //     .then((response) => response.json())
+  //     .then((data) => {
+  //       if (data.message === 'LOGIN_SUCCESS') {
+  //         localStorage.setItem('token', data.token);
+  //         navigate('/threadPost');
+  //       } else if (data.message === 'EMAIL_NOT_FOUND') {
+  //         alert('가입되지 않은 이메일 입니다.');
+  //         navigate('/');
+  //       } else if (data.message === 'WRONG_PASSWORD') {
+  //         alert('비밀번호가 틀렸습니다.');
+  //         navigate('/');
+  //       }
+  //     });
+  // };
+  return (
+    <>
+      <div className="top">
+        <button className="topback" onClick={goToMain}>
+          &#60; 뒤로가기
+        </button>
+
+        <span className="toptext"> 로그인 </span>
+
+        <button className="topadmin" onClick={() => setIsClicked(true)}>
+          관리자 계정
+        </button>
+      </div>
+      <div>
+        <div className="main">
+          <img id="logo" src="/images/gitcat.png" />
+          <div className="maincont">"페이지 간단 소개 글"</div>
+          <img className="maincont" src="/images/social.png" />
+          <div className="social"></div>
+        </div>
+
+        <div className={`login${isClicked ? ' visible' : ''}`}>
```

Comment:
visible 앞에 빈 여백이있네요! login className와 이어지는것을 방지하기 위함이라면
```suggestion
        <div className={`login ${isClicked ? 'visible' : ''}`}>
```
login와 변수사이 자체에 공백을 주셔야 합니다.